src/linux/init/main.cpp

```diff
@@ -3875,6 +3875,63 @@ Return Value:
 
 int WslEntryPoint(int Argc, char* Argv[]);
 
+void EnableDebugMode(const std::string& Mode)
+{
+    if (Mode == "hvsocket")
```

Comment:
We might want to think about how to handle multiple debug modes, maybe comma separated? This is fine for now though since there is only one.